samples/DotNet/Rbac/AzureEventHubsSDK/Program.cs

```diff
@@ -18,79 +18,126 @@ class Program
         static readonly string EventHubNamespace = ConfigurationManager.AppSettings["eventHubNamespaceFQDN"];
         static readonly string EventHubName = ConfigurationManager.AppSettings["eventHubName"];
 
-        static void Main()
+        static int Main()
+        {
+            return MainAsync().GetAwaiter().GetResult();
+        }
+
+        private static async Task<int> MainAsync()
         {
             Console.WriteLine("Choose an action:");
             Console.WriteLine("[A] Authenticate via Managed Identity and send / receive.");
             Console.WriteLine("[B] Authenticate via interactive logon and send / receive.");
             Console.WriteLine("[C] Authenticate via client secret and send / receive.");
             Console.WriteLine("[D] Authenticate via certificate and send / receive.");
```

Comment:
> Console.WriteLine("[D] Authenticate via certificate and send / receive."); [](start = 11, length = 75)

Please add another option for access token. This will return the token from config instead of grabbing it from AAD.

Reply:
i'll give this option to the data sender/receiver EPH sample (separate sample)

Reply:
Or maybe we can skip this one completely.

---
In reply to: [297014177](https://github.com/Azure/azure-event-hubs/pull/444#discussion_r297014177) [](ancestors = 297014177)